docs/porting/libraries.md

```diff
@@ -1,5 +1,153 @@
-# 🔧 Porting your libraries
+# Porting Libraries to .NET Core
 
-> **Note**
-> 
-> This topic hasn’t been written yet! You can track the status of this issue through our public GitHub issue tracker. Learn how you can contribute on GitHub.
+blah blah some intro
```

Comment:
Yes. I'm pretty sure otherwise end up with something quite similar to [this epic obsoletion](https://web.archive.org/web/20130906071245/http://msdn.microsoft.com/en-us/library/system.net.websockets.websocket.isapplicationtargeting45.aspx) message we all missed.